guides/common/modules/proc_configuring-satellite-for-outgoing-emails.adoc

```diff
@@ -5,6 +5,7 @@ To send email messages from {ProjectServer}, you can use either an SMTP server,
 
 .Prerequisites
 
+* Postfix from {RHEL} is the recommended email server to use. Some SMTP servers with anti-spam protection or grey-listing features are known to cause problems, to setup outgoing email with such service either install and configure postfix on {ProjectServer} for relay or use `sendmail` command instead.
```

Comment:
Can you cite a source on this? Note that Debian ships exim by default.

Reply:
The context is: https://bugzilla.redhat.com/show_bug.cgi?id=1911299

Red Hat QA do actually perform regression testing when Satellite is configured via local Postfix. Exim will work fine as long as some anti-spam protection isn't configured. I cannot be specific what exactly can cause issues tho, the purpose of this change is to inform user that it's safest to get an email from Foreman into a proper SMTP agent as soon as possible so it can be delivered properly.